### ISSUE TITLE
Refactor the code to Promises

### ISSUE THREAD

**Issue by jay51** [jay51/fitness-q-a#2]

**Comment by peoray** [comment]
Hey, still actively doing this project? Would love to get in and help on the issues

**Comment by peoray** [comment]
Sure, no problem. Would help in any way I can. Just let me know, then we can schedule time to discuss well

**Comment by jay51** [comment]
If you have time u can ask me anything on discord. This is my username jay#8691

**Comment by peoray** [comment]
Are you resuming work on it? And if yes, I'll need help and questions

**Comment by jay51** [comment]
@peoray yes, I still working on this. as for regrade for your questions, you can ask me on discord ( i can also walk you through the project step by step with video ) or you can ask me here on gh. Also you didn't tell my what you can help with such as design, building the user interface or back end or even DB?
